zio-http-gen-sbt-plugin/src/sbt-test/zio-http-sbt-codegen/dynamic/test

```diff
@@ -0,0 +1,6 @@
+# compile should depend on generated sources,
+# thus we'll check it triggers codegen by inspecting expected .class files
+# compiled from expected generated sources
+> compile
+$ exists target/scala-2.13/classes/gov/uspto/ibd/v1/weeklyarchivedata/SearchWeeklyArchiveData.class
+$ exists target/scala-2.13/classes/gov/uspto/ibd/component/ArchiveDataRecord.class
```

Comment:
nit:
might want to also check for existence of `target/scala-2.13/src_managed/oapi/gov/uspto/ibd/api.json`
Just so it can serve as documentation for users where to expect dynamic openapi files to be found.